.env.development

```diff
@@ -3,3 +3,4 @@ NODE_ENV=development
 BABEL_ENV=development
 NODE_PATH=./openassessment/xblock/static/js/src
 BROWSER=none
+PORT=8080 # port for webpack dev server
```

Comment:
Add a newline to file.

Don't use 8080 if possible, lots of other dev stuff defaults to this port.

Reply:
I changed it to 18001...